src/components/operations/operation-details/ko/runtime/operation-console.ts

```diff
@@ -149,10 +149,25 @@ export class OperationConsole {
 
         await this.loadSubscriptionKeys();
 
+        this.trackOptionsChanges();
         this.updateRequestSummary();
         this.working(false);
     }
 
+    private trackOptionsChanges(): void {
+        if (this.consoleOperation()) {
+            const operation = this.consoleOperation();
+            const queryParams = operation.request.queryParameters();
+            if (queryParams) {
+                queryParams.map(p => p.options.length > 0 && p.value.subscribe(() => this.updateRequestSummary()));
```

Comment:
doesn't this binding above do the same? `data-bind="event: { keyup: $component.updateRequestSummary }`

Reply:
is keyup event triggers on mouse events?

Reply:
no, in this case maybe it's not needed in markup?